visualization/app/codeCharta/util/deltaGenerator.ts

```diff
@@ -98,6 +98,8 @@ export class DeltaGenerator {
 
 			for (const [key, value] of Object.entries(node.attributes)) {
 				node.deltas[key] = -value
+				delete node.attributes[key]
+				// node.attributes[key] = 0
```

Comment:
Setting it to zero seems better as the attribute should exist.

If you do not want to set any attributes (which I definitely would), it would be better to just assign `node.attributes = {}` after the loop. That way it's also faster as deleting properties from an object changes the internal form and the object access becomes slow.

Reply:
If the edge metric is not used in delta mode, we could think about it to remove it.